creator-node/src/fileManager.js

```diff
@@ -88,6 +88,7 @@ async function saveFileToIPFSFromFS ({ logContext }, cnodeUserUUID, srcPath, ipf
  * @param {Array} gatewaysToTry List of gateway endpoints to try
  * @param {String?} fileNameForImage file name if the multihash is image in dir.
  *                  eg original.jpg or 150x150.jpg
+ * @return {Boolean} success indicator
```

Comment:
for clarity i'd change to `true if success, false if error` to make it more explicit